stats/cloud/collector.go

```diff
@@ -461,9 +462,13 @@ func (c *Collector) aggregateHTTPTrails(waitPeriod time.Duration) {
 				aggrData.CalcAverages()
 
 				if aggrData.Count > 0 {
-					logrus.WithFields(logrus.Fields{
-						"http_samples": aggrData.Count,
-					}).Debug("Aggregated HTTP metrics")
+					/*
+						logrus.WithFields(logrus.Fields{
+							"http_samples": aggrData.Count,
+							"ratio":        fmt.Sprintf("%.2f", float64(aggrData.Count)/float64(trailCount)),
+							"t":            time.Since(start),
+						}).Debug("Aggregated HTTP metrics")
+					//*/
```

Comment:
Why not delete this?

Reply:
we probably want to use it at a later point, it's just too noisy for non debugging purposes